contrib/kafka/pkg/reconciler/kafkasource.go

```diff
@@ -87,6 +87,16 @@ func (r *reconciler) Reconcile(ctx context.Context, object runtime.Object) error
 		logger.Error("could not find Apache Kafka source", zap.Any("object", object))
 		return nil
 	}
+	logger.Info("Source bootstrap servers:", zap.Any("BootstrapServers", src.Spec.BootstrapServers))
+	if src.Spec.BootstrapServers == "" {
+		logger.Error("Apache Kafka source missing bootstrapServers", zap.Any("BootstrapServers", src.Spec.BootstrapServers))
+		return nil
+	}
+	logger.Info("Source topics:", zap.Any("Topics", src.Spec.Topics))
+	if src.Spec.Topics == "" {
+		logger.Error("Apache Kafka source missing topics", zap.Any("Topics", src.Spec.Topics))
+		return nil
+	}
```

Comment:
Let me know if more checks should be added or we could better validation?

Reply:
I was wondering... if there is no better `openAPIV3Schema` ? 

/cc @syedriko

Reply:
you should be able to do this by changing the open API schema by adding `topics` in the required list of properties

Reply:
@lionelvillard do you think `bootstrapServers` and `topics` checks could be added to 
https://github.com/knative/eventing-sources/blob/master/contrib/kafka/config/300-kafkasource.yaml

Reply:
will be trying something like this

```
  validation:
    openAPIV3Schema:
      properties:
        apiVersion:
          type: string
        kind:
          type: string
        metadata:
          type: object
        spec:
          properties:
            bootstrapServers:
              type: string
              minLength: 1
            topics:
              type: string
              minLength: 1
            consumerGroup:
              type: string
            serviceAccountName:
              type: string
            sink:
              type: object
          type: object
          required:
              - bootstrapServers
              - topics
```

Reply:
this looks right.